projects/storefrontapp/src/environments/features/my-account/my-account.feature.ts

```diff
@@ -0,0 +1,6 @@
+import { MyAccountModule } from '@spartacus/my-account';
+import { FeatureEnvironment } from '../../models/feature.model';
+
+export const myAccountFeature: FeatureEnvironment = {
```

Comment:
This feels over-engineered, I'm hoping the top level feature module is enough and already a good entry-point to import.